### PR TITLE
fix/development errors

### PR DESCRIPTION
**Pull request by ephraimeg**

### CODE REVIEW

File: livestream/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,60 +1,43 @@
...
     dependencies = [
-        ('auth', '0009_alter_user_last_name_max_length'),
     ]
 
     operations = [
         migrations.CreateModel(
-            name='User',
+            name='Appeal',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```
Comment (jonathantumulak) [inline livestream/migrations/0001_initial.py]: @ephraimeg what happened to these migrations? why are they edited?

Comment (jonathantumulak) [reply livestream/migrations/0001_initial.py]: is this from moving the user model to its own app?

Comment (ephraimeg) [reply livestream/migrations/0001_initial.py]: I deleted all the previous migration files

File: livestream/models.py [hunk 2]
```diff
@@ -55,11 +57,17 @@ def __str__(self):
     def get_description(self):
         return self.detail
 
-    def activate(self):
-        self.status = self.ACTIVE
+    def set_available(self):
+        self.status = Appeal.AVAILABLE
```
Comment (aldnav) [inline livestream/models.py]: Does this work without explicitly saving the instance?

Comment (aldnav) [reply livestream/models.py]:
```
appeal = Appeal.objects.first()
appeal.set_available()
assert appeal.is_active is True
appeal = Appeal.objects.first()  # or appeal.refresh_from_db()
assert appeal.is_active is True
```

Comment (aldnav) [reply livestream/models.py]: @ephraimeg @allenazamoras

Comment (ephraimeg) [reply livestream/models.py]: thanks for pointing it out, I checked and it doesn't change the value unless it is `save()` is called

File: userprofile/serializers.py [hunk 3]
```diff
@@ -19,28 +29,42 @@ class Meta:
         model = User
         fields = ('id', 'username', 'first_name',
                   'last_name', 'gender', 'profile_picture',
-                  'online', 'openappeals', 'closedappeals', 'offers')
+                  'overallrating', 'online', 'openappeals',
+                  'closedappeals', 'offers')
 
-    def get_offers(self, obj):
+    def get_overallrating(self, obj):
```
Comment (aldnav) [inline userprofile/serializers.py]: @ephraimeg @allenazamoras can you check to see if you can do `aggregation` here? 

Ref: https://docs.djangoproject.com/en/2.0/topics/db/aggregation/

Comment (aldnav) [reply userprofile/serializers.py]: Or perhaps a simple order_by rating.

Comment (aldnav) [reply userprofile/serializers.py]: Also check if you can use reverse relation to get ratings from the user instance.

**Pull request merged**